frontend/src/metabase/entities/model-indexes/actions.unit.spec.ts

```diff
@@ -59,12 +77,11 @@ describe("Entities > model-indexes > actions", () => {
         createMockField(),
       ]);
 
-      const mockDispatch = jest.fn();
-      const mockGetState = jest.fn();
-      await updateModelIndexes(model)(mockDispatch, mockGetState);
+      const { dispatch, getState, dispatchSpy, getStateSpy } = setup();
+      await updateModelIndexes(model)(dispatch, getState);
```

Comment:
weird type issue, looks like it matches the signature from a glance. happy to look at it with you if needed.